tutorials/backup-postgresql-pgbackrest-s3/index.mdx

```diff
@@ -0,0 +1,256 @@
+---
+meta:
+  title: Backup PostgreSQL databases with pgBackRest to Object Storage
+  description: How to install pgBackRest on Scaleway Instance for backing up PostgreSQL databases to Object Storage
+content:
+  h1: Backup PostgreSQL databases with pgBackRest to Object Storage
+  paragraph: This page shows how to backup PostgreSQL databases with pgBackRest to Object Storage
+tags: compute networking database sql postgresql pgbackrest backup buckets Object-Storage s3
+steps:
```

Comment:
Oh sorry. I didn't realised that. I will delete it. Thanks. :)

Reply:
I deleted that part in 4c0ddd105ee0305be147ad18c2fae07aa8842989

Reply:
> Hi, thanks for your tutorial ! There are some formatting things to correct so that it displays correctly, I've left comments/suggestions where necessary. Don't hesitate if you have any questions or problems.

Hi, thank you for the review and your time. :) I'm beginning to think that my tutorials cause more problems for you other than helping...